src/pages/physical-count-itemDetails/index.js

```diff
@@ -0,0 +1,633 @@
+import { VertLayout } from 'src/components/Shared/Layouts/VertLayout'
+import { Grow } from 'src/components/Shared/Layouts/Grow'
+import { useContext, useEffect, useState, useRef } from 'react'
+import { RequestsContext } from 'src/providers/RequestsContext'
+import { ResourceIds } from 'src/resources/ResourceIds'
+import { Fixed } from 'src/components/Shared/Layouts/Fixed'
+import { Button, Grid } from '@mui/material'
+import { ControlContext } from 'src/providers/ControlContext'
+import { useResourceQuery } from 'src/hooks/resource'
+import ResourceComboBox from 'src/components/Shared/ResourceComboBox'
+import { SCRepository } from 'src/repositories/SCRepository'
+import FormShell from 'src/components/Shared/FormShell'
+import CustomNumberField from 'src/components/Inputs/CustomNumberField'
+import { useForm } from 'src/hooks/form'
+import * as yup from 'yup'
+import { DataGrid } from 'src/components/Shared/DataGrid'
+import { InventoryRepository } from 'src/repositories/InventoryRepository'
+import { SystemRepository } from 'src/repositories/SystemRepository'
+import { SystemChecks } from 'src/resources/SystemChecks'
+import toast from 'react-hot-toast'
+
+const PhysicalCountItemDe = () => {
+  const { getRequest, postRequest } = useContext(RequestsContext)
+  const { platformLabels } = useContext(ControlContext)
+  const [siteStore, setSiteStore] = useState([])
+  const [controllerStore, setControllerStore] = useState([])
+  const [filteredItems, setFilteredItems] = useState([])
+  const [editMode, setEditMode] = useState(false)
+  const [disSkuLookup, setDisSkuLookup] = useState(false)
+  const [jumpToNextLine, setJumpToNextLine] = useState(false)
+  const [showDefaultQty, setShowDefaultQty] = useState(false)
+  const [disableItemDuplicate, setDisableItemDuplicate] = useState(false)
```

Comment:
disableItemDuplicate is not used so why add the state?

Reply:
for future code modifications